app/assets/stylesheets/application.scss

```diff
@@ -68,3 +80,20 @@ a {
 .nav_conditional_show p {
   margin-bottom: 0rem;
 }
+
+.alert-alert, .alert-notice {
+    color: #170838;
+    background-color: #ff590080;
+    border-color: #ff590080;
+}
+
+.alert {
+    margin-top: 1rem;
```

Comment:
looks like there's additional indentation here compared to other CSS rules above.